Sources/XcodesKit/Path+.swift

```diff
@@ -0,0 +1,8 @@
+import Path
+
+extension Path {
+    static let oldXcodesApplicationSupport = Path.applicationSupport/"ca.brandonevans.xcodes"
```

Comment:
This is probably new to me but what is with the quotes?

Reply:
The `/` is an infix operator that does the same thing as `Path.join`, so the quotes here are just because `ca.brandonevans.xcodes` is a String literal. It would be equivalent to `Path.applicationSupport.join("ca.brandonevans.xcodes")`.